sdcm/cluster.py

```diff
@@ -2776,7 +2776,7 @@ def get_gossip_info(self) -> dict[BaseNode, dict]:
             if line.startswith('SCHEMA:'):
                 schema = line.replace('SCHEMA:', '')
             elif line.startswith('RPC_ADDRESS:'):
-                ip = line.replace('RPC_ADDRESS:', '')
+                ip = ipaddress.ip_address(line.replace('RPC_ADDRESS:', '')).exploded
```

Comment:
I wasn't considering the outout of gossipinfo... 👍🏼

Reply:
I am validating that it won't broke IPv4 test